javaee/api/src/main/java/org/jboss/forge/addon/javaee/servlet/SecurityConstraintDescriptor.java

```diff
@@ -0,0 +1,177 @@
+package org.jboss.forge.addon.javaee.servlet;
+
+import org.jboss.forge.addon.javaee.security.TransportGuarantee;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Descriptor of the security constraint element part of the web.xml
+ *
+ * @author <a href="mailto:ivan.st.ivanov@gmail.com">Ivan St. Ivanov</a>
+ */
+public class SecurityConstraintDescriptor
```

Comment:
Do you mean org.jboss.shrinkwrap.descriptor.api.webapp<release>.SecurityConstraintType? The thing with it is that is Servlet spec version specific. But I want to use this as a transfer object from the command to the facet. And at the same time keep the code in the command release independent (that is why we have the facet implementations). That is why I decided to create an object of my own. Do you have a different idea?